draft-ietf-oauth-sd-jwt-vc.md

```diff
@@ -495,6 +495,8 @@ This section defines Type Metadata that can be associated with a type of a SD-JW
    according to the rules of the type. For example, a Verifier can check
    whether a credential contains all required claims and whether the claims
    are selectively disclosable.
+ * Wallets can use the metadata to display the credential in a way that is
+   consistent with the Issuer's intent.
```

Comment:
but `Type Metadata` is not necessarily or even typically produced or hosted by an issuer...